source/_posts/2016-10-01-we-have-raspberry-image-now.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "We have a Raspberry Pi image now"
+description: "Release of the Rapsberry Pi Image for Home Assistant"
+date: 2016-10-01 01:00:00 -0400
+date_formatted: "October 1, 2016"
+author: Fredrik Lindqvist
+comments: true
+categories: Technology
+---
+
+## {% linkable_title We finally have a Raspberry Pi image! %}
+
+It's been named Hassbian in honor of Raspbian.
+This image comes pre-installed with the dependencies, permissions and so on for Home Assistant.  
+Image is based on the same install instructions as the [Manual installation](/getting-started/installation-raspberry-pi/).  
+It's based on Raspbian Lite and generated with a fork of the same [script](https://github.com/home-assistant/pi-gen) that builds the [official Raspbian images](raspberrypi.org/downloads/raspbian/). Please note that this project has no association with the Raspberry Pi foundation or their projects.  
+
+<!--more-->
+
+On first boot the latest release of Home Assistant will be [installed][/getting-started/installation-raspberry-pi-image.markdown]and can after 3~5 minutes.
+Pre-installed on this image is the MQTT broker [Mosquitto](https://mosquitto.org/), Bluetooth support for none Bluetooth LE and settings for the `homeassistant` user to use the GPIO pins of the Raspberry Pi. Mosquitto is not activated by default.
+
```

Comment:
Well there is Bluetooth support pre-installed but no support for Bluetooth LE since that requires some stranger changes to the python executable or running HA as root.
I'll rewrite it somewhat.